src/app/components/mnoe-organizations-local-list/mnoe-organizations-local-list.html

```diff
@@ -9,6 +9,7 @@
         <tr>
           <th translate>mnoe_admin_panel.dashboard.organization.widget.local_list.search_organizations.table.name</th>
           <th style="width: 100px;" translate>mnoe_admin_panel.dashboard.organization.widget.local_list.search_organizations.table.created_at</th>
+          <th style="width: 130px;" translate>mnoe_admin_panel.dashboard.organization.widget.local_list.search_organizations.table.role</th>
```

Comment:
Doesn't the hardcoded `width` cause issue with responsiveness?

Reply:
Before, the right side (created_at) was fixed width, whereas the left side resized. Decided to keep it this way, as it looks good and is nice and responsive. Also the fixed width makes it so that the x and plus sign will never go onto another line.